WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListFragment.java

```diff
@@ -491,6 +452,33 @@ public void onPostButtonClicked(int buttonType, PostModel post) {
         }
     }
 
+    public void filterPosts(final String searchTerm) {
+        // cancel scheduled search runnable
+        mHandler.removeCallbacks(mSearchRunnable);
+
+        if (!TextUtils.isEmpty(searchTerm)) {
+            showLoadMoreProgress();
+            mSearchTerm = searchTerm;
+            mHandler.postDelayed(mSearchRunnable, SEARCH_DELAY_MS);
+        } else {
+            hideLoadMoreProgress();
+            mRecyclerView.setAdapter(getPostListAdapter());
+        }
+    }
+
+    private void updateAdapter(List<PostModel> posts) {
```

Comment:
We probably should rename this to something related to search since it's only used for that and the `updateAdapter` name is too generic that I was wondering why we weren't using it elsewhere.